src/themes/elife/social-sharers.ts

```diff
@@ -0,0 +1,78 @@
+      url: `https://twitter.com/intent/tweet/?text=${encodedTitle}&url=${encodedDoi}`,
+    },
+    {
+      description:
+        'Email a link to this page (opens up email program, if configured on this system)',
+      imageMarkup:
+        '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M22 4H2C.9 4 0 4.9 0 6v12c0 1.1.9 2 2 2h20c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zM7.25 14.43l-3.5 2c-.08.05-.17.07-.25.07-.17 0-.34-.1-.43-.25-.14-.24-.06-.55.18-.68l3.5-2c.24-.14.55-.06.68.18.14.24.06.55-.18.68zm4.75.07c-.1 0-.2-.03-.27-.08l-8.5-5.5c-.23-.15-.3-.46-.15-.7.15-.22.46-.3.7-.14L12 13.4l8.23-5.32c.23-.15.54-.08.7.15.14.23.07.54-.16.7l-8.5 5.5c-.08.04-.17.07-.27.07zm8.93 1.75c-.1.16-.26.25-.43.25-.08 0-.17-.02-.25-.07l-3.5-2c-.24-.13-.32-.44-.18-.68s.44-.32.68-.18l3.5 2c.24.13.32.44.18.68z"/></svg>',
+      url: `mailto:?subject=${encodedTitle}&body=${encodedDoi}`,
+    },
+    {
+      description: 'Share this page on Reddit',
+      imageMarkup:
+        '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M24 11.5c0-1.65-1.35-3-3-3-.96 0-1.86.48-2.42 1.24-1.64-1-3.75-1.64-6.07-1.72.08-1.1.4-3.05 1.52-3.7.72-.4 1.73-.24 3 .5C17.2 6.3 18.46 7.5 20 7.5c1.65 0 3-1.35 3-3s-1.35-3-3-3c-1.38 0-2.54.94-2.88 2.22-1.43-.72-2.64-.8-3.6-.25-1.64.94-1.95 3.47-2 4.55-2.33.08-4.45.7-6.1 1.72C4.86 8.98 3.96 8.5 3 8.5c-1.65 0-3 1.35-3 3 0 1.32.84 2.44 2.05 2.84-.03.22-.05.44-.05.66 0 3.86 4.5 7 10 7s10-3.14 10-7c0-.22-.02-.44-.05-.66 1.2-.4 2.05-1.54 2.05-2.84zM2.3 13.37C1.5 13.07 1 12.35 1 11.5c0-1.1.9-2 2-2 .64 0 1.22.32 1.6.82-1.1.85-1.92 1.9-2.3 3.05zm3.7.13c0-1.1.9-2 2-2s2 .9 2 2-.9 2-2 2-2-.9-2-2zm9.8 4.8c-1.08.63-2.42.96-3.8.96-1.4 0-2.74-.34-3.8-.95-.24-.13-.32-.44-.2-.68.15-.24.46-.32.7-.18 1.83 1.06 4.76 1.06 6.6 0 .23-.13.53-.05.67.2.14.23.06.54-.18.67zm.2-2.8c-1.1 0-2-.9-2-2s.9-2 2-2 2 .9 2 2-.9 2-2 2zm5.7-2.13c-.38-1.16-1.2-2.2-2.3-3.05.38-.5.97-.82 1.6-.82 1.1 0 2 .9 2 2 0 .84-.53 1.57-1.3 1.87z"/></svg>',
+      url: `https://reddit.com/submit/?title=${encodedTitle}&url=${encodedDoi}`,
+    },
+  ]
+}
+
+const buildSharer = (
+  description: string,
+  imageMarkup: string,
+  url: string,
+  wrapper: Element
+): void => {
+  const link = create('a', {
+    href: url,
+    class: 'social-sharer-link',
+    'aria-label': description,
+    target: '_blank',
+    rel: 'noopener noreferrer',
+  })
+  // Using DOM method innerHTML, not a Thema util method, as a way to get inline SVG text incorporated as an Element
+  link.innerHTML = imageMarkup
```

Comment:
@alex-ketch The requirement here is to be able to use text strings describing SVG images (e.g. [line 15](https://github.com/stencila/thema/pull/154/files#diff-9d2f0a0b8f0331f569f956da45255b4dR15)) to insert SVG elements into the DOM. I couldn't find a `util` method which would do that, hence addressing `innerHTML` directly here. Wondering if there's a more Thema-friendly way to do it?

Reply:
@davidcmoulton you're right, seems like there currently isn't a utility for doing what you need.
Would you mind logging an issue please?

Reply:
The believe that the `create` function already does what you need:

```js
  const link = create('a', {
    href: url,
    class: 'social-sharer-link',
    'aria-label': description,
    target: '_blank',
    rel: 'noopener noreferrer',
  }, create(imageMarkup))
```

I created a PR to document and test that: https://github.com/stencila/thema/pull/155. Let me know if that isn't what you need.

Reply:
Thanks @nokome, just the job. Sorry, I missed that 😅